corehq/apps/app_manager/sql_models.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import absolute_import
```

Comment:
Non-blocking: I'd have named this something like `release_models` or `limited_release_models`, so it explains what the models are, not their implementation details.

Reply:
good point.